source/features/expand-all-hidden-comments.ts

```diff
@@ -0,0 +1,48 @@
+import select from 'select-dom';
+import delegate, {DelegateEvent} from 'delegate-it';
+import features from '../libs/features';
+
+const EXPANDER_SELECTOR = '.ajax-pagination-form';
+const REMAIN_COMMENTS_REGEX = /\s*(\d+) hidden items\s*/;
+
+function handleAltClick(event: DelegateEvent<MouseEvent, HTMLFormElement>): void {
+	if (!event.altKey) {
+		return;
+	}
+
+	const form = event.delegateTarget;
+
+	const buttons = select.all('button', form);
+	const remainCommentsButton = buttons.find(button => REMAIN_COMMENTS_REGEX.test(button.textContent!))!;
+	const [remainComments] = REMAIN_COMMENTS_REGEX.exec(remainCommentsButton.textContent!)!;
```

Comment:
```js
const hiddenItemsCount = Number(form.textContent.replace(/\D+/g, ''));
```

Reply:
`URLSearchParams` requires the value to be type of `string`, so I left it as is.

Reply:
Good point, however you missed the `form` and `hiddenItemsCount` parts. Finding the button is not needed. This is the whole function; you can move the description to the header of the feature, below the imports.

```ts
function handleAltClick(event: DelegateEvent<MouseEvent, HTMLButtonElement>): void {
	if (!event.altKey) {
		return;
	}

	const {form} = event.delegateTarget;
	const hiddenItemsCount = form.textContent!.replace(/\D+g/, '');
	const url = new URL(form.action);
	url.searchParams.set('variables[first]', hiddenItemsCount);
	form.action = url.href;
}

function init(): void {
	delegate('.ajax-pagination-form [type="submit"]', 'click', handleAltClick);
}
```

Also notice that I (just now) changed it to listen to clicks on the button rather than the form, because the form includes the squiggly line next to it and does nothing.

Reply:
Ahh right! This is much cleaner now! 👍

Reply:
You missed the variable name again: `hiddenItemsCount`, not `remainComments`

Reply:
😱 Sorry 😞